src/components/Icon/Icon.module.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../assets/variables.scss";
+
+@mixin icons{
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
+
+.icon{
+  @include icons;
+}
+
+.buttonIcon{
+  @include icons;
+  background: none;
+  color: inherit;
+  border: none;
+  font: inherit;
```

Comment:
some of the properties with `inherit` may not be required, cause they have `inherit` by default